webui/react/src/components/Table.tsx

```diff
@@ -98,9 +97,16 @@ export const expermentDurationRenderer: ExperimentRenderer = (_, record) => {
 };
 
 export const experimentProgressRenderer: ExperimentRenderer = (_, record) => {
-  if (!record.progress) return;
-  return <ProgressBar
+  return record.progress ? <ProgressBar
     percent={record.progress * 100}
-    state={record.state}
-    title={floatToPercent(record.progress, 0)} />;
+    state={record.state} /> : '-';
+};
+
+/* Table Helper Functions */
+
+export const isAlternativeAction = (event: React.MouseEvent): boolean => {
+  const target = event.target as unknown as HTMLElement;
```

Comment:
I think this doesn't need to be cast to `unknown` first. removing that `as` should make it safer.

Reply:
ah you're right, nice!